components/esp_lvgl_port/examples/rgb_lcd/main/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(LV_DEMO_DIR ../managed_components/lvgl__lvgl/demos)
+file(GLOB_RECURSE LV_DEMOS_SOURCES ${LV_DEMO_DIR}/*.c)
```

Comment:
This should no longer be needed. When you enable `CONFIG_LV_USE_DEMO_WIDGETS` in menuconfig, the demo source files will be added to the build

https://github.com/lvgl/lvgl/blob/master/env_support/cmake/esp.cmake#L21